README.md

```diff
@@ -17,8 +17,10 @@ A collection of awesome resources about public speaking. Please read the [contri
 * [Propose, Prepare, Present](http://shop.oreilly.com/product/0636920027096.do) - Free Ebook by **Alistair Croll** (O'Reilly)
 * [Presentation Patterns](http://presentationpatterns.com/) - by **Neal Ford**, **Matthew McCullough**, and **Nate Schutta** (Addison-Wesley)
 * [Presentation Zen Design](http://www.amazon.com/gp/product/0321668790) - by Garr Reynolds. How to design your presentation.
-* [TED Talks: The Official TED Guide to Public Speaking](http://www.amazon.com/d/0544634497/) - by **Chris Anderson** (Houghton Mifflin Harcourt). Start to finish guide to giving powerful talks.
-
+* [TED Talks: The Official TED Guide to Public Speaking](http://www.amazon.com/d/0544634497/) - by **Chris Anderson** (Houghton Mifflin Harcourt). Start to finish guide to giving powerful talk
+ THE WAR OF ART BY STEPHEN PRESSFIELD
+  TALK LIKE TED: THE 9 PUBLIC-SPEAKING SECRETS OF THE WORLD’S TOP MINDS BY CARMINE GALLO
+  CONFESSIONS OF A PUBLIC SPEAKER BY SCOTT BERKUN
```

Comment:
Hi, thanks for your pull request.
Can you please update following the format of the previous link (bullet point, markdown with a link to the book and short description?)
Thanks.